src/parseData.js

```diff
@@ -0,0 +1,55 @@
+const fs      = require('fs');
+const request = require('request');
+const cheerio = require('cheerio');
+
+request('http://www.covidmaroc.ma/Pages/AccueilAR.aspx', function(error, response, html) {
+	if(!error) {
+		var $ = cheerio.load(html);
+
+		var confirmed, deaths, recovered, notConfirmed, date;
+		var json = require('./databackup.json');
+
+		$('#MSOZoneCell_WebPartWPQ1').find('p').each(function(index, el) {
+			var vel = $(el).html();
+			if (index == 3 || index == 4) {
+				vel = $(el).text().trim().replace(/\s\s/g, '');
+				if (index == 3) {
+					confirmed = vel;
+				}
+				else {
+					notConfirmed = vel;
+				}
+			}
+			else if (index == 0) {
+				date = $(el).text().replace(/^\s+/g, '');
+			}
+			else if (index == 1) {
+				vel = $(el).html().replace('&#x200B;<br><span class="ms-rteThemeForeColor-9-4" style="text-decoration: none solid #b10026;">', ' ').replace(/[^0-9. ]/g, '');
+				var tab = vel.split(' ');
+				deaths = tab[1];
+				recovered = tab[0];
+			}
+		});
+
+		$('.ms-rteTable-6').find('tr').each(function(index, el) {
+			if (index > 0 && index <= 12) {
```

Comment:
Why 0 and 12?

Reply:
in 0 you will get 'Région​​ | Nombre de cas confirmé​s'

Reply:
and 12 since there is only 12 regions on the table

Reply:
The 0 makes sense, but 12 doesn't to me, the table is already limited right? why limit it with the index?

Reply:
for extra security xD. since i Don't trust microsoft's aspx

Reply:
it's not needed but when I was writing the code and added the > 0 condition I thought it would make since to limit it.

Reply:
Just my old habit